src/tests/programs/create-program.ts

```diff
@@ -65,5 +58,7 @@ module.exports = {
       .end();
   },
 
-  afterEach,
```

Comment:
is his `afterEach` no longer relevant? how come?

Reply:
we have `after` needing to be defined, to record the results at the end of a test
`afterEach` is called after each step. where it was used before has been taken over by having `after`.